zerver/openapi/zulip.yaml

```diff
@@ -15670,6 +15691,10 @@ paths:
                             description: |
                               Whether clients should display the [number of starred
                               messages](/help/star-a-message#display-the-number-of-starred-messages).
+                          typing_notifications_enabled:
+                            type: boolean
+                            description: |
+                              Whether users should receive typing notifications from other users or not.
```

Comment:
You can leave off the "or not" here. And being more precise: 

"Whether this user is configured to receive typing notifications from other users. The server will only deliver typing notifications events to users who for whom this is enabled."

Reply:
done